content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md

```diff
@@ -8,6 +8,9 @@ content_template: templates/task
 
 {{% capture overview %}}
 {{< feature-state state="alpha" >}}
+
+// TODO(mtaufen): update this doc for v1.11/beta
+
 As of Kubernetes 1.8, the new
```

Comment:
When you update this, please just rely on the feature-state tag above and don't mention the version a second time. You can use the feature-state shortcode like: 

```
{{< feature-state for_k8s_version="v1.11" state="beta" >}}
```